tests/Functional/tests/indented-list/indented-list.html

```diff
@@ -1,7 +0,0 @@
-<p>Testing an indented list:</p>
```

Comment:
This tested indented lists to be parsed as lists. But in fact, reSt doesn't allow any whitespace before the list marker. Such whitespace (indentation) is recognized as a blockquote.